Makefile

```diff
@@ -161,7 +161,7 @@ generate-go: $(CONTROLLER_GEN) $(CONVERSION_GEN) ## Runs Go related generate tar
 		--output-file-base=zz_generated.conversion \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
 	$(CONVERSION_GEN) \
-		--input-dirs=./bootstrap/kubeadm/api/v1alpha2 \
+		--input-dirs=./bootstrap/kubeadm/api/v1alpha3 \
```

Comment:
Wait, is this correct? It looks like the conversion gen annotation is defined on the v1alpha2 types not the v1alpha3 types: https://github.com/kubernetes-sigs/cluster-api/blob/98b4afbeedf5b7d87fbf6f49859675d31bd903f6/bootstrap/kubeadm/api/v1alpha2/doc.go#L17